cli/consts.go

```diff
@@ -17,7 +17,7 @@ var Dev bool = false
 const DefaultTTL uint = 1
 
 // DefaultTimeRemaining for new key requests in minutes
-const DefaultTimeRemaining uint = 60
+const DefaultTimeRemaining uint = 5
```

Comment:
We dropped this to 5 minutes because the default TTL is 1 hour, which means that by default users will *always* be re-prompted to credentials otherwise. This makes that feature actually useful.